_includes/domain-search.html

```diff
@@ -36,6 +36,18 @@
                 </div>
                 `
                 )
+            }
+            else if (response.code === 'error') {
```

Comment:
@rachidatecs looks like Dave's ticket had been sitting up for a while. Should this still get merged or does this conflict with your changes too much?